lua/recall/telescope.lua

```diff
@@ -19,7 +19,7 @@ M.extension = function(opts)
   local function display_entry(entry)
     return entry_layout({
       { config.opts.sign, config.opts.sign_highlight },
-      entry.value.info.file,
+      entry.value.char .. ":" .. entry.value.info.file .. ":" .. entry.value.info.pos[2]
```

Comment:
Instead of indexing into `info.pos`, can we destructure? I believe this works in Lua:

```lua
local _, lnum, col = unpack(entry.info.pos)
```

And then this becomes more readable:
```suggestion
      entry.value.char .. ":" .. entry.value.info.file .. ":" .. lnum
```